src/servant/tt_002/faerie.h

```diff
@@ -33,32 +33,31 @@ typedef enum {
 #define ROOM_SPECIAL_STATE_SUS_HINT2 9
 #define ROOM_SPECIAL_STATE_SUS_HINT3 11
 
-// SFX ids
-#define NA_VO_FAERIE_INTRO_LIFE 0x472    // "Thank you for giving me life"
-#define NA_VO_FAERIE_INTRO_COMMAND 0x473 // "Your word is my command"
-#define NA_VO_FAERIE_LETS_GO 0x476       // "Let's go!"
-#define NA_VO_FAERIE_FOLLOW 0x479        // "I'll follow you anywhere"
-// "There is something funny about this wall"
-#define NA_VO_FAERIE_WALL_HINT 0x47A
-// "This is really suspicious looking"
-#define NA_VO_FAERIE_SUSPICIOUS_HINT 0x48A
-#define NA_VO_FAERIE_MIST_HINT 0x48E // "If only you could transform into mist"
-// "Some animals can live in complete darkness"
-#define NA_VO_FAERIE_DARKNESS_HINT 0x492
-#define NA_VO_FAERIE_HEALING 0x885      // "Healing!"
-#define NA_VO_FAERIE_POTION 0x886       // "Potion"
-#define NA_VO_FAERIE_REGENERATION 0x887 // "Regeneration"
-#define NA_VO_FAERIE_NO_MEDICINE 0x888  // "No medicine left"
-#define NA_SE_FAERIE_HAI 0x889
-#define NA_SE_FAERIE_YEAH 0x88A
-#define NA_SE_FAERIE_HA1 0x88B
-#define NA_SE_FAERIE_HA2 0x88C
-#define NA_SE_FAERIE_AH 0x88D
-#define NA_SE_FAERIE_UHH 0x88E
-#define NA_SE_FAERIE_UHH2 0x88F
-#define NA_SE_FAERIE_OHH 0x890
-#define NA_VO_FAERIE_OH_NO 0x891      // "OH NOOO!"
-#define NA_VO_FAERIE_ARE_YOU_OK 0x892 // "Are you ok?" possibly unreachable code
+typedef enum {
+    FAERIE_INTRO_LIFE = 0x472, // "Thank you for giving me life"
```

Comment:
As far as I understand we're combining two different things here. The lower numbers are XA audio while the higher ones are sound effects. The higher ones should be part of https://github.com/Xeeynamo/sotn-decomp/blob/0c25f09d9747a5521f282fead949d4d7ca202ea5/include/sfx.h#L564

Reply:
Originally, when I did the Faerie, I had the SFX ids in the sfx header. 
I think we had a conversation about it here:
https://github.com/Xeeynamo/sotn-decomp/pull/1909

I don't think we really ever game to any hard conclusion.  I think this is a design philosophy question really, ie, do we want the keys to be in the places that they are used or in a single enum.  I don't particularly have a problem with either, but I'm quite sure that they keys here are used only in the Faerie (possibly the japan only fairy as well?).  My opinion would be that it makes more sense to have the keys that are used by one entity in that entity's code and have the shared keys in the sfx header.  I do realize this makes things a bit harder to sus out what is shared and what isn't when we are only part way through the work though.  That being said, I believe all of the keys that are not loaded from the servant specific binary (ie, all the ones under 0x600), are only used inside those servants anyway.

I don't really have a problem with either way, but we should probably see what @Xeeynamo thinks?

Reply:
AFAIK the only actual evidence we have is the shifted IDs in MAD. We don't have any info about servants. https://github.com/Xeeynamo/sotn-decomp/blob/0c25f09d9747a5521f282fead949d4d7ca202ea5/include/sfx.h#L264
The shifted IDs there do suggest it was an enum originally though or at least 0x630-0x6DB was

Reply:
From what I have seen thus far, there are no collisions in the servant keys. At least there isn't in the servants we currently have (unknown if true about sword).  

I think they *could* be all in a single enum.  It's really more a question of if we want to organize it that way.

Reply:
Sounds good.  I will move the keys that are shared into sfx.h (those are all they keys under 0x600 btw) and will keep the keys that are loaded from the VB/VH in the servant file.

I'll fix this tonight when I get home from work, then this should be good to merge.